src/test/java/de/mediathekview/mserver/crawler/basic/TopicUrlDTOTest.java

```diff
@@ -1,31 +1,31 @@
-package de.mediathekview.mserver.crawler.orf;
+package de.mediathekview.mserver.crawler.basic;
 
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.junit.Assert.assertThat;
 import org.junit.Test;
 
-public class OrfTopicUrlDTOTest {
+public class TopicUrlDTOTest {
   
   @Test
   public void equalsTestSameUrlAndTheme() {
-    OrfTopicUrlDTO dto1 = new OrfTopicUrlDTO("Thema", "Url");
-    OrfTopicUrlDTO dto2 = new OrfTopicUrlDTO("Thema", "Url");
+    TopicUrlDTO dto1 = new TopicUrlDTO("Thema", "Url");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Thema" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)